sdk/storagepool/Azure.ResourceManager.StoragePool/tests/Scenario/DiskPoolTests.cs

```diff
@@ -0,0 +1,71 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+using System.Threading.Tasks;
+
+using ResourceGroup = Azure.ResourceManager.Resources.ResourceGroup;
+using DiskPoolSku = Azure.ResourceManager.StoragePool.Models.Sku;
+using Azure.ResourceManager.StoragePool.Models;
+using Azure.ResourceManager.Resources.Models;
+using Azure.Core;
+
+namespace Azure.ResourceManager.StoragePool.Tests
+{
+    [TestFixture(true)]
```

Comment:
Can you remove this attribute? This will block our test framework to run the test both in sync and async mode.